autoware_map_msgs/package.xml

```diff
@@ -5,12 +5,14 @@
   <version>1.7.0</version>
   <description>Autoware map messages package.</description>
 
-  <maintainer email="koji.minoda@tier4.jp">kminoda</maintainer>
-  <maintainer email="mfc@leodrive.ai">M. Fatih Cırıt</maintainer>
+  <maintainer email="mfc@autoware.ai">Mete Fatih Cırıt</maintainer>
   <maintainer email="yutaka.kondo@tier4.jp">Yutaka Kondo</maintainer>
+  <maintainer email="ryohsuke.mitsudome@tier4.jp">Ryohsuke Mitsudome</maintainer>
 
   <license>Apache License 2.0</license>
 
+  <author email="koji.minoda@tier4.jp">Koji Minoda</author>
```

Comment:
```suggestion
  <author>Koji Minoda</author>
```

Reply:
Done in https://github.com/autowarefoundation/autoware_msgs/pull/132/commits/4a1e4f3239dffea71ad93f4859102bc20d28ae9a 👍